### ISSUE TITLE
iOS Schedule Does Not Begin Tracking

### ISSUE THREAD

**Issue by DomzF** [transistorsoft/cordova-background-geolocation-lt#1102]
## Your Environment
* Plugin version: 3.4.2
* Platform: iOS (old)
* OS version: 11.4
* Device manufacturer / model: iPhone 6S
* Cordova version (`cordova -v`): 8.1.2
* Cordova platform version (`cordova platform ls`): 5.0.1
* Plugin config provided to `#ready`:
```javascript <!-- syntax-highlighting:  paste your code below -->
{
"activityType":1,
"elasticityMultiplier":1,
"disableStopDetection":false,
"didLaunchInBackground":false,
"autoSync":true,
"desiredAccuracy":-1,
"stopTimeout":5,
"minimumActivityRecognitionConfidence":70,
"useSignificantChangesOnly":false,
"stopDetectionDelay":10,
"autoSyncThreshold":2,
"batchSync":true,
"disableElasticity":false,
"enableTimestampMeta":false,
"locationAuthorizationAlert":{"settingsButton":"Settings","cancelButton":"Cancel","titleWhenNotEnabled":"Background location is not enabled","instructions":"To use background location, you must enable '{locationAuthorizationRequest}' in the Location Services settings","titleWhenOff":"Location services are off"}
,"url":"REDACTED",
"stopOnStationary":false,
"isFirstBoot":false,
"startOnBoot":true,
"locationAuthorizationRequest":"Always",
"httpTimeout":30000,
"stopOnTerminate":false,
"lastLocationAuthorizationStatus":3,
"maxBatchSize":100,
"maxRecordsToPersist":4000,
"activityRecognitionInterval":10000,
"isMoving":false,
"trackingMode":1,
"stationaryRadius":25,
"preventSuspend":false,
"extras": 
{"user_id":40725},
"disableMotionActivityUpdates":false,
"odometer":90171.01331233507,
"locationTimeout":60
,"params: {"utc_offset":-300}
,"method":"POST",
"logMaxDays":3,
"geofenceInitialTriggerEntry":true,
"iOSHasWarnedLocationServicesOff":false,
"disableLocationAuthorizationAlert":false,
"distanceFilter":10,
"headers":{"X-TOKEN":"REDACTED"},
"heartbeatInterval":60,
"httpRootProperty":"location",
"disableAutoSyncOnCellular":false,
"desiredOdometerAccuracy":100,
"locationTemplate":"",
"locationsOrderDirection":"ASC",
"persistMode":2,
"stopAfterElapsedMinutes":-1
,"schedule":["2 00:00-23:59","3 00:00-23:59","4 00:00-23:59","5 00:00-23:59","6 18:15-23:59","7 00:00-23:59","1 00:00-23:59"],
"authorization":{},
"schedulerEnabled":true,
"debug":false,
"pausesLocationUpdatesAutomatically":false,
"geofenceProximityRadius":1000,
"geofenceTemplate":"",
"maxDaysToPersist":2,
"encrypt":false,
"logLevel":2,
"enabled":true
}
```

## Expected Behavior
Despite having the application closed, I believe the plugin should have began tracking automatically this (monday) morning. I was running an android device in parallel and it performed wonderfully.

I do understand this is a very old device with an outdated OS, so should either be the source of the issue I would like to be able to tell my users to upgrade.

## Actual Behavior
The plugin did not begin tracking until I opened the application this morning at 8AM EST.

## Steps to Reproduce

1. Set up a schedule for the plugin.
2. Execute startSchedule
3. Terminate the front end of the application

## Context
See if tracking would start even though the front end of the application was closed.

## Debug logs

<details>
	<summary>Logs</summary>

```<!-- Syntax highlighting:  DO NOT REMOVE -->
2020-03-01 13:11:05.936 ⚠️-[TSHttpService flush:] No network connection

2020-03-01 13:12:40.079 ⚠️-[TSHttpService flush:] Busy with previous request

2020-03-01 16:51:54.542 ⚠️-[LocationManager locationManager:didUpdateLocations:] <IGNORED> (location-age -788.7 s)

2020-03-01 16:51:54.619 ⚠️-[TSHttpService flush:] Busy with previous request

2020-03-02 08:34:35.769 ⚠️-[TSHttpService flush:] Busy with previous request

2020-03-02 08:34:40.274 ⚠️-[TSHttpService flush:] Busy with previous request
```

</details>

As you can see there is a large gap in the logs!

Thanks, love the product.

**Comment by DomzF** [comment]
I realize after checking back you need a debug log, I'll update the issue when I can reproduce it.

**Comment by DomzF** [comment]
Cannot reproduce this issue specifically, please see the new issue I am posting.
